common/lexer/flex_lexer_adapter.h

```diff
@@ -107,6 +113,21 @@ class FlexLexerAdapter : private CodeStreamHolder, protected L, public Lexer {
     }
   }
 
+  // Skip n characters from the begining of the token
+  void YYSkip(size_t skip) {
+    last_token_.text = absl::string_view(
+        last_token_.text.begin() + skip, last_token_.text.length() - skip);
```

Comment:
This is the same as:

https://github.com/abseil/abseil-cpp/blob/1a02b7a2054c24f900dab796edb812f9260b51a6/absl/strings/string_view.h#L331

Maybe name this something like `SkipPrefix` to make it clearer what is skipped?

Reply:
Right, changed the implementation and renamed, thanks!